mmv1/products/integrations/Client.yaml

```diff
@@ -89,6 +90,13 @@ properties:
       Indicates if sample workflow should be created along with provisioning.
     immutable: true
     ignore_read: true
+    deprecation_message: "`create_sample_workflows` is deprecated and will be removed in a future major release. Use `create_sample_integrations` instead."
+  - !ruby/object:Api::Type::Boolean
+    name: 'createSampleIntegrations'
```

Comment:
Could you please [add a conflicts field](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#conflicts) to stop a user specifying both fields in the same resource block?

Reply:
Done